source/includes/_install_android.md

```diff
@@ -0,0 +1,129 @@
+    <version>2.4.7</version>
+</dependency>
+```
+
+If you use Maven, you can include this library as a dependency.
+
+## Using Gradle
+```xml
+compile 'com.wootric:wootric-sdk-android:2.4.7'
+```
+
+ Add the following to your app’s `build.gradle` file, inside the dependencies section.
+
+# Presenting the survey
+WootricSDK task is to present a fully functional survey view with just a few lines of code.
+
+## Step 1. Add permissions
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+```
+Add the internet permissions to the `AndroidManifest.xml` file.
+
+
+## Step 2. Add ProGuard rules
+```ProGuard
+-keepattributes *Annotation*, Signature
+
+##== Wootric ==
+-keep class com.wootric.** { *; }
+
+##== Retrofit ==
+-keep class retrofit.** { *; }
+-keepclassmembernames interface * {
+    @retrofit.http.* <methods>;
+}
+```
+
+Add the following to your ProGuard rules file.
+
+## Step 3. Initialize Wootric
+```java
+import com.wootric.androidsdk.Wootric;
+
+// Inside your method
+Wootric wootric = Wootric.init(this, <YOUR_CLIENT_ID>, <YOUR_CLIENT_SECRET>, <YOUR_ACCOUNT_TOKEN>);
+
+// TODO: The current logged in user's email address.
+wootric.setEndUserEmail("nps@example.com");
+// TODO: The current logged in user's sign-up date as a Unix timestamp.
+wootric.setEndUserCreatedAt(1234567890);
+
+// Use only for testing
+wootric.setSurveyImmediately(true);
+
+wootric.survey();
+```
+First import the SDK into your Activity of choosing. Then configure the SDK with your client ID, secret and account token.
+
+Sign in at [wootric.com](https://www.wootric.com/) if you haven't already. If you just signed up on the Wootric homepage, you will be taken directly to an installation page. If you’re a returning visitor, click on the “Settings" button near the top right of the page. Navigate to the [Wootric Installation Guide](https://app.wootric.com/install) and you will see a unique **account_token** for you to use.
+
+The **client_id** and **client_secret** can also be found on your Settings page, on the [API section](https://app.wootric.com/account_settings/edit#!/api).
+
+To display the survey just use `wootric.survey();`. If the user is eligible (this check is built in the method) the SDK will show the survey.
+
+
+## Step 4. Customize the Survey
+
+> Comment out the line `wootric.setSurveyImmediately(true)` when you are ready for production. Alternatively, leave the line in the code for testing purposes or to survey the customer upon every visit to a view.
+
+```java
+import com.wootric.androidsdk.Wootric;
+import com.wootric.androidsdk.objects.WootricCustomThankYou;
+
+Wootric wootric = Wootric.init(this, "your_client_id", "your_client_secret", "NPS­-xxxxxxxx");
+wootric.setEndUserEmail("nps@example.com");
+wootric.setEndUserCreatedAt(1234567890);
+
+// Use only for testing
+wootric.setSurveyImmediately(true);
+
+// Customization
+WootricCustomThankYou customThankYou = new WootricCustomThankYou();
```

Comment:
Screenshot of customization page otherwise my instinct says that I would have to code all it by myself while we prefer that they do it from our dashboard.